### ISSUE TITLE
Not enough mesh peers in attestation subnets

### ISSUE THREAD

**Issue by dapplion** [ChainSafe/lodestar#3348]
**Describe the bug**

Contabo-6 with 16 validators should have peers in at least 32 subnets at all times. However the behavior is very erratic. Sometimes it has peers in a single subnet, then it has peers in all subnets. I added a new chart with equation
```
count(lodestar_gossip_mesh_peers_by_beacon_attestation_subnet > 0)
```
which shows the total count of subnets with 1 peer or more

![Screenshot from 2021-10-08 18-25-34](https://user-images.githubusercontent.com/35266934/136583738-b5cf3419-ec74-45e9-abc5-ec08d212b788.png)

![Screenshot from 2021-10-08 18-22-34](https://user-images.githubusercontent.com/35266934/136583748-7d809d37-cc80-4605-a477-8bf86bcbf399.png)

**Expected behavior**

Stay connected to a stable number of subnets

**Comment by twoeths** [comment]
should be fixed by #3403

**Comment by dapplion** [comment]
> should be fixed by #3403

I'm not sure, it will fix it. We need more data

**Comment by dapplion** [comment]
Duplicate of https://github.com/ChainSafe/lodestar/issues/3555
